torch/csrc/Dtype.cpp

```diff
@@ -30,6 +31,10 @@ PyObject* THPDtype_is_floating_point(THPDtype* self, PyObject* noargs) {
   }
 }
 
+PyObject* THPDtype_itemsize(THPDtype* self, PyObject* noargs) {
```

Comment:
from the docs:

`It should return a new reference on success or NULL with a set exception on failure.`

Do you need to box this in HANDLE_TH_ERRORS + END_HANDLE_TH_ERRORS?